docs/en_US/guides/blocking-updates.md

```diff
@@ -46,8 +51,10 @@ By removing update blocking, you may unintentionally update to a higher firmware
 
 1. Open the Settings application
 1. Go to `General` -> `Profile`
-  - This may also be called `Profile and Device Management`
-3. Tap on the tvOS Beta Profile
-1. Tap "Remove Downloaded Profile"
+    - This may also be called `Profile and Device Management` or `VPN & Device Management`
+1. Tap `tvOS 17 Beta Software Profile` -> `Remove Downloaded Profile`
```

Comment:
The rest is obvious, but I'll just mention that since I fixed the line above's indenting, I was able to replace the `3.` with a `1.` on this line, without affecting the site